core/src/main/java/com/spotify/ffwd/output/CoreOutputManager.java

```diff
@@ -59,7 +61,7 @@
     private static final Logger log = LoggerFactory.getLogger(CoreOutputManager.class);
     private static final String[] KEYS_NEVER_TO_DROP = {"ffwd-java", "ffwd-java.ffwd-java"};
     private static final int HYPER_LOG_LOG_PLUS_PRECISION_NORMAL = 14;
-    private static final int HYPER_LOG_LOG_PLUS_PRECISION_SPARSE = 25;
+    private static final int HYPER_LOG_LOG_PLUS_PRECISION_SPARSE = 5;
```

Comment:
why lower this?

Reply:
if you look in the comparison the engineer did in his blog post the others even don't work

Reply:
ah i see, i didnt equate r with that variable off the top of my head

Reply:
I've renamed them to reflect different library args

Reply:
nice thanks :)